.appveyor.yml

```
@@ -123,8 +127,19 @@ test_script:
         $env:phpunit_config = "tests\appveyor\$($env:db).$($env:driver).appveyor.xml"
       }
 
-      vendor\bin\phpunit -c $($env:phpunit_config)
+      if ($env:coverage -eq "yes") {
+        vendor\bin\phpunit -c $($env:phpunit_config) --coverage-clover clover.xml
+      } else {
+        vendor\bin\phpunit -c $($env:phpunit_config)
+      }
 
       if ($LastExitCode -ne 0) {
         $host.SetShouldExit($LastExitCode)
       }
+
+after_test:
+  - ps: |
+      Invoke-WebRequest -Uri 'https://codecov.io/bash' -OutFile codecov.sh
+      $env:APPVEYOR="true"
+      $env:CI="true"
+      bash codecov.sh -f "clover.xml"
```

Hi, Tom from Codecov here. I'm not sure this will fix the issue, but I have a sneak suspicion it might.
```suggestion
      bash codecov.sh -f clover.xml
```

---

Thanks, let's try!

---

Looks like it doesn't help…

---

FWIW, I successfully switched to Codecov and PCOV on Appveyor in https://github.com/diff-sniffer/diff-sniffer/pull/34 and https://github.com/diff-sniffer/diff-sniffer/pull/35.

---

Thanks it was helpful!
1. I remove the env variables, didn't help
2. I switched to multiline syntax, and got an error message I did not see before: `Invoke-WebRequest -Uri 'https://codecov.io/bash' -OutFile codecov.sh
'Invoke-WebRequest' is not recognized as an internal or external command,
operable program or batch file`.
3. I used the same download command as you did, and now it works!